docs/getting-started/maintenance.md

```diff
@@ -146,4 +146,368 @@ In output look for following line:
+3:01PM INF Discovered new snapshot format=1 hash="��z�~�Pc���ϸ��2\x18������ݻPңG��z" height=1809000 module=statesync
+3:01PM INF Discovered new snapshot format=1 hash="Ĝ+���W���!\x0e�\x1b_����gu��x%��Ew�" height=1808800 module=statesync
+3:01PM INF Discovered new snapshot format=1 hash="��/\x0e\v\x16�7E�\x1c�\x1a%��\n��\tɮ��%z\x105祅l" height=1808600 module=statesync
+3:01PM INF Discovered new snapshot format=1 hash="\x1e��\x16�Z�R�\x1f��kq\x1e\t`�\v��37�\r�\x15��q" height=1808400 module=statesync
+3:01PM INF Discovered new snapshot format=1 hash="��\x10�V�Z�Z{8~k�sx�)/\"T�\u007f+��\t���\x06^" height=1808300 module=statesync
+3:01PM INF Discovered new snapshot format=1 hash="n4�ʹ/��xژI*�L�\x1bf�ҝ1#\x10�>c\x05A{\x11c" height=1807500 module=statesync
+...
+..
+```
+
+Shortly after finding the best peers and repopulating the address book, you will begin to download the blockchain at a lightening speed compared to full sync!
+Remember, this puts your database at a truncated state.  You will only be able to query information from the machine you apply these settings to, only at the block height that you initiated state sync at.
+
+### Troubleshooting
+
+Q: It continues to look for snapshots and doesn't ever provide me one
+A: Check your seeds/peers. After reseting the the data dir and addressbook, you must redownload it.  If you fail to do so, it will not start state sync.
+
+Q: Validator set is Nil??
+A: See above.
+
+Q: It's still not working and my seeds/peers are correct.
+A: Try another `unsafe-reset-all` and restart.  If that does not work, change `pruning = nothing` in `app.toml` and restart the service.  State Sync will sometimes complain if you try to prune everything when it's first applying a state sync snapshot.
+
+If you are still running into issues, come to the discord and share your log errors in the #Validator-operators channel.  It is possible for a snapshot to be stuck in creation that would prevent delivery of snapshots for statesync.
+
+## Database Pruning
+
+Doing less than the recommended settings here is not recommended nor supported!
+
+Pruning your database to acheive the best minimal result will require that you first **delete** what you have and then use **state sync** to catch back up.
+These are advanced instructions, continue only if you know what you are doing.
+
+Lets also get an idea of how much disk space this has saved us.  It's important to keep track of these metrics so you can understand how well your node is performing!
+```bash
+df -h
+du -h -d 1 /home/vidulum/.vidulum/data
+```
+
+::: tip TIP:
+Before proceeding **backup all configuration files**
+:::
+
+
+### Prepare to prune your database
+
+Backup your blockchain data before continuing in the event the unexpected happens.
+The following command will **DELETE YOUR DATABASE** for the blockchain.  Continue if you only know what you are doing.
+
+As your `vidulum` user:
+```bash
+vidulum unsafe-reset-all
```

Comment:
Should be vidulumd unsafe-reset-all

Reply:
Live working copy to view: http://erialosofastora.asuscomm.com:8080/